src/main/java/org/terasology/quests/examples/BookQuestSystem.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2015 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.quests.examples;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.books.logic.BookComponent;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterMode;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.logic.common.ActivateEvent;
+import org.terasology.tasks.Quest;
+import org.terasology.tasks.Task;
+import org.terasology.tasks.components.QuestComponent;
+import org.terasology.tasks.events.BeforeQuestEvent;
+import org.terasology.tasks.systems.QuestSystem;
+import org.terasology.registry.In;
+
+/**
+ * Handles quest attached to book entity.
```

Comment:
Maybe expand ever so slightly? Book-borned quests you can acquire by using a book, what is meant to happen if the book leaves your hand or the inventory, etc.

Reply:
I will add that functionality in a separate pr.